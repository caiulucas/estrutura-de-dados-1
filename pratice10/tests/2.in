Defining the genre
==================
 
A novel is a long, fictional narrative which describes intimate human experiences. The novel in the modern era usually makes use of a literary prose style, and the development of the prose novel at
this time was encouraged by innovations in printing, and the introduction of cheap paper, in the 15th century.

The present English (and Spanish) word for a long work of prose fiction derives from the Italian novella for "new", "news", or "short story of something new", itself from the Latin novella, a singular
noun use of the neuter plural of novellus, diminutive of novus, meaning "new".[note 1] Most European languages use the word "romance" (as in French, Dutch, Russian, Slovene, Serbo-Croatian, Romanian,
Danish, Swedish and Norwegian "roman"; German "Roman"; Portuguese "romance" and Italian "romanzo") for extended narratives.
