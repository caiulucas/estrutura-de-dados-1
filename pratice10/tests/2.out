8
a 5
and 5
as 1
at 1
by 1
century 1
cheap 1
croatian 1
danish 1
defining 1
derives 1
describes 1
development 1
diminutive 1
dutch 1
encouraged 1
english 1
era 1
european 1
experiences 1
extended 1
fiction 1
fictional 1
for 3
french 1
from 2
genre 1
german 1
human 1
in 4
innovations 1
intimate 1
introduction 1
is 1
italian 2
itself 1
languages 1
latin 1
literary 1
long 2
makes 1
meaning 1
modern 1
most 1
narrative 1
narratives 1
neuter 1
new 3
news 1
norwegian 1
note 1
noun 1
novel 3
novella 2
novellus 1
novus 1
of 8
or 1
paper 1
plural 1
portuguese 1
present 1
printing 1
prose 3
roman 2
romance 2
romanian 1
romanzo 1
russian 1
serbo 1
short 1
singular 1
slovene 1
something 1
spanish 1
story 1
style 1
swedish 1
th 1
the 12
this 1
time 1
use 3
usually 1
was 1
which 1
word 2
work 1
